### PR TITLE
ETFE-1572: update GTM container

### PR DESCRIPTION
**Pull request by r-melvin**

### CODE REVIEW

File: conf/application.conf
Comment (MJCallahanPage) [inline conf/application.conf:1]: There are some commented out lines above relating to this that can either be used or deleted.

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]
Minor tweak

File: conf/application.conf
Comment (r-melvin) [reply conf/application.conf:1]: should be sorted now

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢 🚀

**Pull request merged**